### ISSUE TITLE
i think hrnet+ocrnet will make your models more perfect

### ISSUE THREAD

**Issue by panker98** [JanMarcelKezmann/TensorFlow-Advanced-Segmentation-Models#5]
i think hrnet+ocrnet will make your models more perfect

**Comment by JanMarcelKezmann** [comment]
You are right the hrnet+ocrnet is definitive one of the strongest semantic segmentation models.
I will consider an implementation in the coming weeks.

**Comment by rhalherrero** [comment]
Also interested in the hrnet+ocrnet implementation. Any progress?

**Comment by JanMarcelKezmann** [comment]
Yes, I am currently testing and if everything works well I will upload it by the end of this weekend.
